src/journal-remote/journal-remote-main.c

```diff
@@ -53,6 +53,11 @@ static bool arg_trust_all = false;
 static bool arg_trust_all = true;
 #endif
 
+static uint64_t arg_max_use = 0;
+static uint64_t arg_max_size = 0;
+static uint64_t arg_n_max_files = 0;
+static uint64_t arg_keep_free = 0;
```

Comment:
As you can see the comment in journal-file.h, the default should be `UINT64_MAX`.

Reply:
done